src/leap/common/events/txclient.py

```diff
@@ -122,7 +124,6 @@ def _run_callback(self, callback, event, content):
         callback(event, *content)
 
     def shutdown(self):
-        TxZmqClientComponent.shutdown(self)
```

Comment:
See https://github.com/smira/txZMQ/blob/master/txzmq/factory.py#L44
Since I'm calling the registerForShutdown on the factory, and the factory keeps track of the connections, I see no need of doing this ourselves.